src/assets/svgs/headset.ts

```diff
@@ -0,0 +1,5 @@
+export default `
+<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M25.0325 12.2477V11.0347C25.0325 8.10811 23.8699 5.30139 21.8005 3.23199C19.7311 1.16258 16.9244 0 13.9978 0C11.0712 0 8.26449 1.16258 6.19508 3.23199C4.12568 5.30139 2.9631 8.10811 2.9631 11.0347V12.2477C1.98263 12.5826 1.1527 13.2543 0.620859 14.1435C0.0890179 15.0326 -0.110265 16.0816 0.0584429 17.1039C0.227151 18.1261 0.752916 19.0554 1.54226 19.7265C2.3316 20.3976 3.33336 20.7671 4.36944 20.7692H5.35123C5.5523 20.7692 5.74513 20.6893 5.88731 20.5471C6.02949 20.4049 6.10937 20.2121 6.10937 20.011V12.7822C6.10937 12.5811 6.02949 12.3883 5.88731 12.2461C5.74513 12.1039 5.5523 12.0241 5.35123 12.0241H4.47937V11.0347C4.47937 8.51025 5.4822 6.0892 7.26725 4.30415C9.0523 2.5191 11.4733 1.51627 13.9978 1.51627C16.5222 1.51627 18.9433 2.5191 20.7283 4.30415C22.5134 6.0892 23.5162 8.51025 23.5162 11.0347V12.0241H22.6443C22.4433 12.0241 22.2504 12.1039 22.1083 12.2461C21.9661 12.3883 21.8862 12.5811 21.8862 12.7822V20.0224C21.8873 20.1618 21.9268 20.2981 22.0004 20.4165C22.074 20.5348 22.1787 20.6306 22.3032 20.6934C21.709 21.7533 20.9192 22.691 19.9757 23.4568C19.7327 22.9749 19.3587 22.5714 18.8967 22.2924C18.4348 22.0135 17.9035 21.8705 17.3639 21.8798H16.1357C15.3686 21.8808 14.6331 22.1858 14.0903 22.7278C13.5475 23.2699 13.2416 24.005 13.2396 24.7721V26.133C13.2361 26.1645 13.2361 26.1963 13.2396 26.2278C13.236 26.258 13.236 26.2885 13.2396 26.3187C13.2396 26.5198 13.3195 26.7126 13.4617 26.8548C13.6039 26.997 13.7967 27.0769 13.9978 27.0769C16.09 27.0743 18.1384 26.4776 19.9047 25.3562C21.6709 24.2348 23.0824 22.6349 23.9749 20.7426C24.9782 20.6623 25.9231 20.2383 26.6502 19.5424C27.3774 18.8464 27.8423 17.921 27.9665 16.9221C28.0908 15.9233 27.8668 14.9121 27.3323 14.0592C26.7978 13.2063 25.9855 12.5639 25.0325 12.2401V12.2477ZM4.59309 13.5403V19.2643H4.36944C3.6104 19.2643 2.88245 18.9627 2.34572 18.426C1.809 17.8893 1.50747 17.1613 1.50747 16.4023C1.50747 15.6433 1.809 14.9153 2.34572 14.3786C2.88245 13.8419 3.6104 13.5403 4.36944 13.5403H4.59309ZM14.7559 25.5303V24.7721C14.7569 24.4068 14.9027 24.0569 15.1614 23.7989C15.42 23.541 15.7704 23.3961 16.1357 23.3961H17.3639C17.6572 23.379 17.9478 23.4608 18.1891 23.6285C18.4304 23.7962 18.6083 24.0401 18.6944 24.321C17.4868 25.0094 16.1418 25.4224 14.7559 25.5303ZM23.6261 19.2529H23.4025V13.5403H23.6261C24.3852 13.5403 25.1131 13.8419 25.6498 14.3786C26.1866 14.9153 26.4881 15.6433 26.4881 16.4023C26.4881 17.1613 26.1866 17.8893 25.6498 18.426C25.1131 18.9627 24.3852 19.2643 23.6261 19.2643V19.2529Z" fill="#1F2C9B"/>
```

Comment:
It's probably a good idea to remove the `fill`s from this and set the fill on the icon itself.